Araali Tools/goshelly-server-helm/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
not meant to be committed - all the auto-generated stuff. Put it into .gitignore